### PR TITLE
Add prioritize_on attribute to triagebot

### PR DESCRIPTION
**Pull request by spastorino**
r? @Mark-Simulacrum

### CODE REVIEW

**Review by LeSeulArtichaut — APPROVED** [review]
Seems good to go for me... Once we land the feature in triagebot though 😜

File: triagebot.toml [hunk 1]
```diff
@@ -36,4 +36,5 @@ label = "ICEBreaker-Cleanup-Crew"
 
 [prioritize]
 label = "I-prioritize"
+prioritize_on = ["regression-from-stable-to-beta", "regression-from-stable-to-nightly"]
```
Comment (Mark-Simulacrum) [inline triagebot.toml:39]: Hm, probably stable-to-stable too?

Comment (spastorino) [reply triagebot.toml:39]: We're not currently prioritizing all of those right now, maybe we could start slowly starting to do so. cc @pnkfelix

Comment (spastorino) [reply triagebot.toml:39]: Actually, let's add those this will fire the request for prioritization for new ones. We can slowly update the old ones.

Comment (spastorino) [reply triagebot.toml]: Fixed.

File: triagebot.toml [hunk 2]
```diff
@@ -36,4 +36,6 @@ label = "ICEBreaker-Cleanup-Crew"
 
 [prioritize]
 label = "I-prioritize"
+prioritize_on = ["regression-from-stable-to-stable", "regression-from-stable-to-beta", "regression-from-stable-to-nightly"]
+priority-labels = "P-*"
```
Comment (Mark-Simulacrum) [inline triagebot.toml:40]:
```suggestion
priority_labels = "P-*"
```

**Review by Mark-Simulacrum — APPROVED** [review]
r=me with nit fixed and commits squashed

**Pull request merged**